ios/browser/api/ads/brave_ads.h

```diff
@@ -149,8 +149,7 @@ OBJC_EXPORT
 
 /// Get inline content ad for the given dimensions
 - (void)inlineContentAdsWithDimensions:(NSString*)dimensions
-                            completion:(void (^)(BOOL success,
-                                                 NSString* dimensions,
+                            completion:(void (^)(NSString* dimensions,
                                                  InlineContentAdIOS*))completion
```

Comment:
This is no longer a valid definition as you are possibly passing nil, so the 2nd arg has to be nullable (`InlineContentAdIOS _Nullable*`)

Reply:
Fixed